materials/r-databases.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+element: notes
+title: dplyr with Databases
+language: R
+---
+
+> Remember to
+>    
+> * download [`portal_mammals.sqlite`](https://ndownloader.figshare.com/files/2292171).
+> * open `portal_mammals.sqlite` in SQLite Manager. 
+> * make sure the copy you are going to use in class does not have the `SpeciesCounts` table or view.
+
+* We've already seen briefly how to work with databases using `dplyr`.
+* Let's get more familiar with the `dplyr` context.
+* There are other database management systems libraries (e.g., `DBI`, `RSQLite`)
```

Comment:
`dplyr` is just using these two libraries behind the scenes so the phrasing here is a little confusing. I might say something like "`dplyr` wraps the low level database management libraries to make it easy to interact with databases"